spec/cancan/model_adapters/active_record_adapter_spec.rb

```diff
@@ -237,101 +237,84 @@ class User < ActiveRecord::Base
     end
 
     it 'has false conditions if no abilities match' do
-      expect(@ability.model_adapter(Article, :read).conditions).to eq("'t'='f'")
+      expect(@ability.model_adapter(Article, :read).database_records.to_sql.strip.squeeze(' '))
+        .to eq(%q(SELECT "articles".* FROM "articles" WHERE (1 == 0)))
     end
 
     it 'returns false conditions for cannot clause' do
       @ability.cannot :read, Article
-      expect(@ability.model_adapter(Article, :read).conditions).to eq("'t'='f'")
+      expect(@ability.model_adapter(Article, :read).database_records.to_sql.strip.squeeze(' '))
```

Comment:
extract `.to_sql.strip.squeeze(' ')` into a method `to_sql(database_records)` and reuse it